{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "17.7.4",
            "versionStartIncluding": "17.7.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "17.8.2",
            "versionStartIncluding": "17.8.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "17.7.4",
            "versionStartIncluding": "17.7.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "17.8.2",
            "versionStartIncluding": "17.8.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:community:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "17.6.5",
            "versionStartIncluding": "16.11.0",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:gitlab:gitlab:*:*:*:*:enterprise:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "17.6.5",
            "versionStartIncluding": "16.11.0",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@gitlab.com",
      "ID": "CVE-2025-1198"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An issue discovered in GitLab CE/EE affecting all versions from 16.11 prior to 17.6.5, 17.7 prior to 17.7.4, and 17.8 prior to 17.8.2 meant that long-lived connections in ActionCable potentially allowed revoked Personal Access Tokens access to streaming results."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-613"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "GitLab Issue #511477",
          "refsource": "",
          "tags": [
            "Broken Link"
          ],
          "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/511477"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "NETWORK",
        "availabilityImpact": "NONE",
        "baseScore": 5.3,
        "baseSeverity": "MEDIUM",
        "confidentialityImpact": "LOW",
        "integrityImpact": "NONE",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
        "version": "3.1"
      },
      "exploitabilityScore": 3.9,
      "impactScore": 1.4
    }
  },
  "lastModifiedDate": "2025-08-06T18:50Z",
  "publishedDate": "2025-02-13T02:15Z"
}